packages/foam-vscode/src/utils.ts

```diff
@@ -162,6 +165,20 @@ export function isSome<T>(value: T | null | undefined | void): value is T {
  *
  * @param value The object to verify
  */
-export function isNone<T>(value: T | null | undefined | void): value is null | undefined | void {
+export function isNone<T>(
+  value: T | null | undefined | void
+): value is null | undefined | void {
   return value == null;
 }
+
+export async function focusNote(notePath: string, isNewNote: boolean) {
```

Comment:
yeah, this makes sense as a utility function.
I know it's unrelated to this PR, but as we are officially making this a utility fn what do you think of making the semantics to be "utility-like"? 
We could rename `isNewNote` with `moveCursorToEnd` or something like that, so it's easier for clients of the fn to understand the result of the parameter value

Reply:
Resolved!